main.go

```diff
@@ -287,17 +289,31 @@ func apply(configFile string) (ctrl.Options, config.Configuration) {
 	}
 	if err != nil {
 		setupLog.Error(err, "unable to load the config")
-		os.Exit(1)
+		return options, cfg, err
+	}
+
+	if cfg.Integrations != nil {
+		var invalidFrameworks []string
+		for _, framework := range cfg.Integrations.Frameworks {
+			if _, found := jobframework.GetIntegration(framework); !found {
+				invalidFrameworks = append(invalidFrameworks, framework)
+			}
+		}
+		if len(invalidFrameworks) > 0 {
```

Comment:
semantically , we are interested to know if the slice is empty. From a performance point of view there could be a penalty but since this not on a performance critical path, it makes more sense to me checking the length.

Reply:
Previously from @alculquicondor, I heard that we don't distinguish the empty slice from the nil slice in the kueue project, and we generally verify if the slice is nil in this case. However, since I may remember incorrectly, I would let me know what @alculquicondor thinks.

Reply:
Indeed, we don't distinguish between `nil` and empty slice in the kubernetes (not just kueue) project. For that reason, `len(x) > 0` is preferred, because `len(nil) == 0`